be/src/exec/vectorized/analytor.cpp

```diff
@@ -383,16 +382,18 @@ void Analytor::update_window_batch(int64_t peer_group_start, int64_t peer_group_
 }
 
 void Analytor::reset_window_state() {
-    SCOPED_TIMER(_compute_timer);
+    // DO NOT put timer here because this function will be used frequently,
+    // timer will cause a sharp drop in performance
     for (size_t i = 0; i < _agg_fn_ctxs.size(); i++) {
         _agg_functions[i]->reset(_agg_fn_ctxs[i], _agg_intput_columns[i],
                                  _managed_fn_states[0]->mutable_data() + _agg_states_offsets[i]);
     }
 }
 
 void Analytor::get_window_function_result(size_t frame_start, size_t frame_end) {
+    // DO NOT put timer here because this function will be used frequently,
+    // timer will cause a sharp drop in performance
     DCHECK_GT(frame_end, frame_start);
-    SCOPED_TIMER(_compute_timer);
```

Comment:
Could we compute this `_compute_timer ` for every chunk, not every row?

Reply:
Not necessary, this time will be recored in `push_chunk` timer or `pull_chunk` timer